OpenTrons/opentrons-api
# feat(app): update historical protocol run table and drawer

Closes [AUTH-453](https://opentrons.atlassian.net/browse/AUTH-453)

# Overview

Update HistoricalProtocolRun component and expanded HistoricalProtocolRunOffsetDrawer to accommodate protocol files and new designs.

# Test Plan

- navigate to a device details page and scroll down to Recent Protocol Runs
- verify that `HistoricalProtocolRun` component matches new designs
- verify that clicking anywhere along the run item expands the files + offset drawer
- verify that `Histori
<img width="668" alt="Screenshot 2024-06-20 at 9 41 01 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/b64c8923-cbba-411e-9573-d45c22fadff9">
calProtocolRunOffsetDrawer` component matches new designs, including for no file/offset data
<img width="665" alt="Screenshot 2024-06-20 at 9 40 01 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/d9b16894-04c2-40c5-aa77-930114efc564">
<img width="685" alt="Screenshot 2024-06-20 at 9 38 59 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/76e357d3-09cc-4bd1-acf1-433830e42d73">
<img width="690" alt="Screenshot 2024-06-20 at 9 38 01 AM" src="https://github.com/Opentrons/opentrons/assets/47604184/62365321-eb5b-4f8d-b03d-447033994741">

# Changelog
- add new translation keys
- update `HistoricProtocolRun` to match designs, including addition of "Files" column
- expand `HistoricProtocolRunDrawer` to display files associated with run

# Review requests

auth js

# Risk assessment

low

[AUTH-457]: https://opentrons.atlassian.net/browse/AUTH-457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[AUTH-453]: https://opentrons.atlassian.net/browse/AUTH-453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
